app/controllers/application_controller.rb

```diff
@@ -25,7 +25,11 @@ def admin_signed_in?
   end
 
   def current_claim
-    @current_claim ||= Claim.find(session[:claim_id]) if session.key?(:claim_id)
+    @current_claim ||= current_claim_from_session
```

Comment:
Instead of assigning directly to `@current_claim` in the `new` and `create` controller actions, we could do this:

```suggestion
    @current_claim ||= current_claim_from_session || Claim.new(eligibility: StudentLoans::Eligibility.new)
```

Reply:
I wondered about that, but will that not always give us a new claim if there's not a claim_id present in the session? It'll also mean we have less control when we come to support Maths and Physics too.

Reply:
ah that's fair, I guess it would complicate things a little when Maths and Physics is introduced...